packages/vscode-extension/src/webview/components/ReplayOverlay.tsx

```diff
@@ -274,6 +274,25 @@ export default function ReplayOverlay({
   // shifting the time a bit here such that it displays the final value properly despite using Math.floor
   const timeSec = Math.floor(time - startTime + 0.05);
   const timeFormat = `${Math.floor(timeSec / 60)}:${(timeSec % 60).toString().padStart(2, "0")}`;
+
+  useEffect(() => {
+    function handleEvents(e: MouseEvent | KeyboardEvent) {
+      e.preventDefault();
+      e.stopPropagation();
+    }
+
+    document.addEventListener("mousedown", handleEvents, true);
+    document.addEventListener("mouseup", handleEvents, true);
```

Comment:
why is this needed here and not needed in other UI items that displays over phone-wrapper?